Cargo.toml

```diff
@@ -32,7 +32,8 @@ tokio = { version = "1.24.2", default-features = false, features = [ "sync", "rt
 zeroize = { version = "1.5.7", default-features = false, features = [ "zeroize_derive" ] }
 
 [target.'cfg(target_family = "wasm")'.dependencies]
-gloo-timers = { version = "0.2.6", default-features = false, features = [ "futures" ] }
+wasm-timer = { version = "0.2.5", default-features = false }
```

Comment:
Are we sure about this? The author themself recommend against using it. What couldn't we do with `gloo-timers` that is supported by `rustwasm` ?

Reply:
HMm no, it was conflicting earlier but not anymore. ill revert 👍

Reply:
Yea gloo_timers was !Send, but thats no longer relevant 🥳